app/src/main/res/values/strings.xml

```diff
@@ -126,4 +126,8 @@
     <string name="title_choose_audio_file">Choose audio file</string>
     <string name="title_del_comment">Delete</string>
     <string name="title_edit_comment">Edit</string>
+
+    <string name="title_option_post">OptionPost</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_option_post` appears to be unused
 Unused resources make applications larger and slow down builds.